src/VerifyU2FController.js

```diff
@@ -62,14 +74,16 @@ function (Okta, FormController, FormType, FooterSignout, Q, HtmlErrorMessageView
             var deferred = Q.defer();
             u2f.sign(appId, factorData.challenge.nonce, registeredKeys, function (data) {
               self.trigger('errors:clear');
+              var rememberDevice = !!self.get('rememberDevice');
```

Comment:
should this be in the `else` block?

Reply:
yes, you're right. It's better in the else block.